src/lib/pages/workflow-query.svelte

```diff
@@ -63,26 +82,31 @@
       <p>{translate('workflows.no-workers-failure-message')}</p>
     </div>
   {:then types}
-    <div class="flex justify-between">
-      <div class="flex items-center gap-4">
-        <Select
-          id="query-select"
-          label={translate('workflows.query-type')}
-          bind:value={queryType}
-          data-testid="query-select"
-        >
-          {#each types as value}
-            <Option {value}>{value}</Option>
-          {/each}
-        </Select>
-        <Button
-          on:click={() => query(queryType)}
-          leadingIcon="retry"
-          loading={isLoading}
-        >
-          {translate('common.refresh')}
-        </Button>
-      </div>
+    <div class="flex items-end justify-between gap-2 max-sm:flex-wrap">
+      <Card class="flex w-full flex-col gap-2 xl:w-2/3">
+        <div class="flex flex-col gap-1">
+          <PayloadInput bind:input label={translate('workflows.query-args')} />
```

Comment:
I'd probably make this 'Query Arg' since we only allow a single argument and not multiple